### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by isidroarrieta** [conveyal/gtfs-data-manager#100]
In the installation instructions it is said "cd into the public/ directory and run component build -dc..." 

But I can't find any component instruction should I have something already installed?

**Comment by ffsiren** [comment]
'npm install component' installs the required command

**Comment by goroshevsky** [comment]
component is no longer supported, after running command `build -dc` I have error  
`fatal : no remote found for dependency "jashkenas/backbone@1.3.3". Visit http://component.github.io/troubleshooting for help.`
This is caused by removing file `component.json` in [jashkenas](https://github.com/jashkenas/backbone) repository 

Also, after command `./activator run` are present unresolved dependencies
`[warn]     :: commons-io#commons-io;2.0.1: configuration not found in commons-io#commons-io;2.0.1: 'master(compile)'. Missing configuration: 'compile'. It was required from com.github.mpeltonen#sbt-idea;1.5.1 compile`

**Comment by Warra07** [comment]
@vadimgoroshevsky yep i guess everyone has the same problem now, i'd like to know if there is any solution
